app/views/pages/home.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="banner" style="background-image: linear-gradient(rgba(0,0,0,0.4),rgba(0,0,0,0.4)), url(https://images.unsplash.com/photo-1515552726023-7125c8d07fb3?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=967&q=80);">
   <div class="container">
-    <h1>Air adventure</h1>
-    <p>Share unbelievable adventures with passionate people !</p>
+    <h1>Experience the adventure of your life</h1>
```

Comment:
@tomapapp --> How about "Live more" ? something more aspirational?

Reply:
I think the user should be able to know what the page is about from the first sentence. So it should provide some kind of explanation what the app does, that is why i left it relatively explicit. But i am flexible.